src/background/stores/base-store.ts

```diff
@@ -1,10 +1,10 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 import { Store } from '../../common/flux/store';
-import { IBaseStore } from '../../common/istore';
+import { BaseStore } from '../../common/istore';
 import { StoreNames } from '../../common/stores/store-names';
 
-export abstract class BaseStore<TState> extends Store implements IBaseStore<TState> {
+export abstract class BaseStoreImpl<TState> extends Store implements BaseStore<TState> {
```

Comment:
I'm wondering if we should update the file name too... it should only affect the files that are already part of this PR, right?

Reply:
I think so. I will check and update this PR.

Reply:
changed it to `base-store-impl.ts`